test/meta-colony.js

```diff
@@ -243,8 +261,8 @@ contract("Meta Colony", accounts => {
       expect(parentId).to.be.zero;
     });
 
-    it("should NOT be able to add a new root global skill", async () => {
-      await checkErrorRevert(metaColony.addGlobalSkill(0), "colony-invalid-skill-id");
+    it.skip("should NOT be able to add a new root global skill", async () => {
```

Comment:
Why skipped?

Reply:
There's no such thing as the root global skill any more. Or are you asking why skipped, and not deleted?

Reply:
We should just delete it really.